crates/swc_common/src/source_map.rs

```diff
@@ -1281,12 +1281,17 @@ impl SourceMap {
             let mut col = max(chpos, linechpos) - min(chpos, linechpos);
 
             if let Some(orig) = &orig {
-                if let Some(token) = orig.lookup_token(line, col) {
```

Comment:
There's actually (what I consider to be) a bug in `lookup_token`, where it'll return an a mapping span on a previous line for spans that:
- extend beyond the last mapped line
- lines that don't have a mapping
- unmapped columns on lines with a mapping

I tried to get them to fix it in https://github.com/getsentry/rust-sourcemap/pull/53, but they rejected it. This breaks with Mozilla's source map behavior, because lines endings are supposed to split the mapping from any previous one.

Anyways, the "fix" is to do:

```rust
orig.lookup_token(line, col).filter(|t| t.get_dst_line() == line)
```

Is this the bug you're trying to fix with the `find`?

Reply:
Yes, exactly. Thank you!

Reply:
I'm not sure about the reason, but it didn't work.